test/facade_converter_test.ts

```diff
@@ -25,9 +25,11 @@ function getSources(str: string): {[k: string]: string} {
     'angular2/traceur-runtime.d.ts': traceurRuntimeDeclarations,
     'angular2/src/di/forward_ref.d.ts': `
         export declare function forwardRef<T>(x: T): T;`,
+    'angular2/src/facade/async.d.ts': `
```

Comment:
Maybe it'd be nice to do a negative test for one of these, e.g. have a different file that defines Observable and see that it doesn't get renamed?